server/src/visitors/YmlFunctionVisitor.ts

```diff
@@ -38,7 +38,7 @@ export class YmlFunctionVisitor extends YmlBaseVisitor {
 
         if (!this.isMethodInstanciation(this.functionName)) {
             const func = new YmlFunction(this.functionName, this.uri);
-            func.enrichWith(node.field(), null);
+            func.enrichWith(node.field(), connection);
```

Comment:
Change in the middle of nowhere? xD

Reply:
In some very rare cases, there was sometimes exceptions thrown because of this null.

Reply:
:woman_shrugging: I hope this `connection` thingy makes sense.

Reply:
It is used to send an error log entry to the client when an exception occurs.